src/vs/workbench/parts/extensions/electron-browser/extensions.contribution.ts

```diff
@@ -203,9 +203,15 @@ Registry.as<IConfigurationRegistry>(ConfigurationExtensions.Configuration)
 		type: 'object',
 		properties: {
 			'extensions.autoUpdate': {
-				type: 'boolean',
-				description: localize('extensionsAutoUpdate', "Automatically update extensions"),
-				default: true,
+				type: 'string',
+				enum: ['checkAndInstall', 'check', 'off'],
+				enumDescriptions: [
+					localize('extensionsAutoUpdateCheckAndInstall', "Install extension updates automatically in the background."),
+					localize('extensionsAutoUpdateCheck', "Check for extension updates and mark extensions with available updates as outdated in the extensions view."),
+					localize('extensionsAutoUpdateOff', "No checks are made automatically for extension updates. You can still manually check for updates using the `Extensions: Check for Updates` command.")
```

Comment:
How is `"extensions.autoUpdate": "off"` implies that extension update checks will not be made? How about having a new setting `extension.autoCheckUpdates`?

Reply:
A new setting would definitely make things more simple. Will give that a go.